pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -90,9 +90,17 @@ var updateDeploymentAndWait = k8sutil.UpdateDeploymentAndWait
 func (c *Cluster) Start() error {
 	logger.Infof("start running mgr")
 
-	dashboardPort := dashboardPortHttps
-	if c.cephVersion.Name == cephv1.Luminous {
-		dashboardPort = dashboardPortHttp
+	var dashboardPort int
+	if c.dashboard.Port == 0 { // select default ports
+		if c.cephVersion.Name == cephv1.Luminous {
+			dashboardPort = dashboardPortHttp
+		} else {
+			dashboardPort = dashboardPortHttps
+		}
+	} else if c.dashboard.Port > 0 {
+		dashboardPort = c.dashboard.Port
```

Comment:
Could you update the unit tests to check these default ports and that the port is used from the crd when set?